src/main/java/com/commercetools/emailprocessor/email/EmailProcessor.java

```diff
@@ -102,4 +108,19 @@ int callApiEndpoint(final String customObjectId, final TenantConfiguration tenan
         }
         return responseCode;
     }
+
+    String encrypt(final String value, final String encryptionKey, final int cipherMode) {
```

Comment:
also, annotations on params? and on return value of method, could it return null?

Reply:
also, it should be static right?

Reply:
88e8e25